xml/admin_ceph_upgrade.xml

```diff
@@ -1109,6 +1109,22 @@ mds standby for name = mds.456-another-mds
     <link xlink:href="https://documentation.suse.com/suse-distribution-migration-system/1.0/single-html/distribution-migration-system/"/>
     to find both general and detailed information about DMS.
    </para>
+    <para>
```

Comment:
I'm also tempted to put this in its own procedure with a title such as "Before you Begin" - any thoughts? I do think it is clear enough as it is, but if anyone sees that as being a good way to call it out.

Reply:
I agree. There will definitely be multiple things to check before starting the upgrade process. For example, we will need to emphasize (as always), that the upgrade must not be started until all the SES6 nodes are fully patched.

Reply:
Good point, I don't believe we do a decent job of highlighting that. I'll add the new section :)

Reply:
(If we forgot that one, we'd be in deep trouble, because then customers on unpatched SES6 would commence the upgrade process and it would blow up, and then they'd point to the documentation and say: "nothing here says we need to patch first!")

Reply:
Story of my life?